NitroxServer/Communication/NetworkingLayer/LiteNetLib/LiteNetLibServer.cs

```diff
@@ -0,0 +1,147 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using LiteNetLib;
+using LiteNetLib.Utils;
+using NitroxModel.DataStructures;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets;
+using NitroxServer.ConfigParser;
+using NitroxServer.GameLogic;
+using NitroxServer.GameLogic.Entities;
+
+namespace NitroxServer.Communication.NetworkingLayer.LiteNetLib
+{
+    public class LiteNetLibServer : NitroxServer
+    {
+        private const string CONNECTION_KEY = "nitrox";
+        private readonly NetManager server;
+        private readonly EventBasedNetListener listener;
+        private readonly NetPacketProcessor netPacketProcessor = new NetPacketProcessor();
+
+        public LiteNetLibServer(PacketHandler packetHandler, PlayerManager playerManager, EntitySimulation entitySimulation, ServerConfig serverConfig) : base(packetHandler, playerManager, entitySimulation, serverConfig)
+        {
+            netPacketProcessor.SubscribeReusable<WrapperPacket, NetPeer>(OnPacketReceived);
+            listener = new EventBasedNetListener();
+            server = new NetManager(listener);
+        }
+        public override void Start()
+        {
+            Log.Info("Using LiteNetLib as networking library");
+
+            listener.PeerConnectedEvent += PeerConnected;
+            listener.PeerDisconnectedEvent += PeerDisconnected;
+            listener.NetworkReceiveEvent += NetworkDataReceived;
+            listener.NetworkReceiveUnconnectedEvent += OnNetworkReceiveUnconnected;
+            listener.ConnectionRequestEvent += OnConnectionRequest;
+
+            server.DiscoveryEnabled = true;
+            server.UnconnectedMessagesEnabled = true;
+            server.UpdateTime = 15;
+            server.UnsyncedEvents = true;
+            server.Start(portNumber);
+
+            isStopped = false;
+        }
+
+        public override void Stop()
+        {
```

Comment:
You're overriding `Stop` but you aren't calling the base code so now it won't do anything. Is this intended?